lib/ASTGen/Package.swift

```diff
@@ -74,6 +75,12 @@ let package = Package(
       path: "Sources/LLVMJSON",
       swiftSettings: swiftSetttings
     ),
+    .target(
+      name: "_CompilerRegexParser",
+      dependencies: [],
+      path: "_RegexParser_Sources",
```

Comment:
I don't see this symlink any more, does it need restoring?

Reply:
Forgot to `git add` 👍